### ISSUE TITLE
Automatically upload APKs to Google Play Alpha

### ISSUE THREAD

**Issue by rafaeltoledo** [rafaeltoledo/social-app#16]

**Comment by rafaeltoledo** [comment]
Blocked by #18

**Comment by rafaeltoledo** [comment]
As we now have App Distribution integrated on Firebase, maybe makes sense to upload merged PRs into develop into Firebase, and PRs merged into master to Google Play
